src/System.Collections/src/System.Collections.csproj

```diff
@@ -26,9 +28,15 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == 'net46'">
-    <TargetingPackReference Include="mscorlib" />
-    <TargetingPackReference Include="System" />
-    <TargetingPackReference Include="System.Core" />
+    <TargetingPackReference Include="mscorlib">
+      <InProject>false</InProject>
```

Comment:
Lets try to fix this by adding this metadata in a common place in the dir.targets or somewhere.